types/ramda/index.d.ts

```diff
@@ -1192,6 +1194,7 @@ declare namespace R {
          * original function's arguments list. In some libraries this function is named `applyLeft`.
          */
         partial(fn: (...a: any[]) => any, ...args: any[]): (...a: any[]) => any;
+        partial<T>(fn: (...a: any[]) => T, ...args: any[]): (...a: any[]) => T;
```

Comment:
This overload does not have associated documentation. Consider copying the documentation from the preceding overload.